msal/src/cache/java/com/microsoft/identity/client/BaseTokenCacheItem.java

```diff
@@ -31,13 +31,13 @@
 abstract class BaseTokenCacheItem {
 
     @SerializedName("client_id")
-    final String mClientId;
+    String mClientId;
```

Comment:
I'll need time to investigate into it. We should finalize everything if we can and of course reduce reflections. I'll get back to the PR on Monday.